ddl/table_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"testing"
 
 	. "github.com/pingcap/check"
```

Comment:
We have a sanity check that after migrate a file, there shouldn't be a dependency to 'pingcap/check'. Please figure out how to remove it.

Reply:
```testTableInfoWithPartition```, ```testTableInfoWithPartitionLessThan``` and so on use ```pingcap/check```. but those functions are used by other test files. so I think it will be removed with other test file migrate.

Reply:
Move them to ```util_test.go```.